front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/StorageServiceSupport.java

```diff
@@ -61,13 +61,13 @@
   public StorageServiceSupport(ObjectType objectType,
                                StorageService service,
                                Scheduler scheduler,
-                               int refreshIntervalMs,
+                               long refreshIntervalMs,
                                Registry registry) {
     this.objectType = objectType;
     this.service = service;
     this.scheduler = scheduler;
     this.refreshIntervalMs = refreshIntervalMs;
-    if (refreshIntervalMs >= HEALTH_MILLIS) {
+    if (refreshIntervalMs >= getHealthMillis()) {
```

Comment:
Delegating to `getHealthMillis()` allows for a DAO to override the healthy interval check (ie. Entity Tags can go longer).